apps/theme/components/Group/Group.tsx

```diff
@@ -54,8 +55,9 @@ export const Group = ({
       )}
 
       <div className={cl(classes.colors)}>
-        {colorNumbers.map((colorNumber, index) => {
-          const { number, hex } = colorScale[colorScheme][colorNumber - 1];
+        {colorNames.map((colorName, index) => {
+          const { number, hex } =
+            colorScale[colorScheme][colorMetadata[colorName].number - 1];
           const color: Color = {
             ...getColorMetadataByNumber(number),
             number,
```

Comment:
Now that we are using acutal color names is this code still correct? Looks like we are picking the color before the one we passed to the group, instead of passed color.

Reply:
oh nvm, its picking from the themeinfo array, which has generated colors. 😩 

Maybe we should refactor `generateColorScale` to return an object with colorName keys instead 🤔